src/createReducer.test.ts

```diff
@@ -47,4 +46,57 @@ test('createReducer should create a redux reducer that updates the state on upda
     expect(newState.d).not.toBe(defaultState.d);
     expect(newState.d.e).toBe(3);
     expect(defaultState.d.e).toBe(2);
+});
+
+test('createReducer should create a redux reducer and an UPDATE_ACTION should update the state and a RESET_ACTION reset to default', () => {
+    const reducer = createReducer(defaultState);
+    const nonSupportedActionType = 'NON_SUPPORTED_ACTION';
+    const updateActionType = ACTION_PREFIX + 'UPDATE';
+    const resetActionType = ACTION_PREFIX + 'RESET';
```

Comment:
Deze action type komt mogelijk niet overeen met de werkelijke. Het is beter voor deze tests om de `createAction()` functie te gebruiken ipv handmatig hier actions te creeren. Dit geldt eigenlijk ook voor de tests hierboven. Dan kunnen we daar in de toekomst als nodig zonder problemen het format van de action veranderen.